.github/workflows/ci-build.yml

```
@@ -47,6 +47,19 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pnpm-store-
 
+      - name: Inject environment variables
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)